packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.spec.js

```diff
@@ -185,4 +185,54 @@ describe('TaskNavButtonsContainer', function () {
       expect(createDefaultAnnotationSpy.notCalled).to.be.true
     })
   })
+
+  describe('#onSubmit', function () {
+    let wrapper
+    let completeClassificationSpy
+    let createDefaultAnnotationIfThereIsNoneSpy
+    let onSubmitSpy
+    const preventDefaultSpy = sinon.spy()
+
+    before(function () {
+      completeClassificationSpy = sinon.spy()
+      createDefaultAnnotationIfThereIsNoneSpy = sinon.spy(TaskNavButtonsContainer.wrappedComponent.prototype, 'createDefaultAnnotationIfThereIsNone')
```

Comment:
Rather than stub the internal component methods here, I'd mock the `classification` and `createDefaultAnnotation` props, so that I can test whether the state of the classification changes as expected.